src/app/containers/NewStoryPromo/PromoBase/Headings.styles.jsx

```diff
@@ -0,0 +1,9 @@
+/* All the baese components needed to build a problem, following the atom molecule pattern 
+    Should I reuse the Promo components in component?
+*/
```

Comment:
And these components could live in the components directory so they could be reused across different pages? Maybe for now they can live alongside the article page, we are considering moving the existing components in the `components` directory to legacy soon as part of the component standards work

Reply:
Bit more background on our plans to move components: WSTEAMA-110